.travis.yml

```diff
@@ -63,6 +65,8 @@ script:
         case "$TRAVIS_JDK_VERSION" in
           "oraclejdk8")
             mvn verify -P dev -Dguava.version=$GUAVA_VERSION ;;
+          "openjdk6")
+            mvn verify -Dmaven.test.skip -Dguava.version=$GUAVA_VERSION ;;
           *)
```

Comment:
Yes. That way we are sure that once we test java9 for example we do not forget and use the default rule.